### ISSUE TITLE
Error in vectorization of logit function

### ISSUE THREAD

**Issue by nthaydt** [Hocking-Lab/cocanal-turtles#11]
commit: 6273ccd30cca281d440027710296561c7e977daf
path: Code/BehavioralModel.R

The error seems to occur around line 37 in the model ->  logit(p0[i, j, k, t]) <- alpha0[k, t] + (alpha2*C[i,k]) + eta[i,k]

I'm having trouble inserting the Cik matrix into the logit function and keep getting the following error:
 Error in checkForRemoteErrors(lapply(cl, recvResult)) : 
  3 nodes produced errors; first error: RUNTIME ERROR:
Invalid vector argument to ilogit 

JAGS doesn't seem to accept the vector provided with the logit function. Creating my own logit function was mentioned as a possible fix.

- Natalie

**Comment by djhocking** [comment]
Fixed the indexing associated with alpha2. **However**, I think rather than by individual and day it should just be by individual at most.

**Comment by nthaydt** [comment]
Yeah, I agree with that. I think making behavior vary by day is too much...

Sent from my iPhone

On Feb 28, 2019, at 12:16 PM, Daniel J. Hocking <notifications@github.com<mailto:notifications@github.com>> wrote:


Fixed the indexing associated with alpha2. However, I think rather than by individual and day it should just be by individual at most.
